cirq/vis/heatmap_test.py

```diff
@@ -0,0 +1,227 @@
+# Copyright 2019 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for Heatmap."""
+
+import string
+
+import numpy as np
+import pytest
+
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+
+from cirq.devices import grid_qubit
+from cirq.vis import heatmap
+
+
+class TestHeatmap:
```

Comment:
we haven't tended to use test classes.

Instead to do the setup I would suggest:
```@pytest.fixture(scope="session", autouse=True)```
to rune the setup once or if you need it once before each method you can use a different scope.

Reply:
Done.

The axes fixture is passed into every test that needs it.